src/index.js

```diff
@@ -1,12 +1,14 @@
 import 'babel-polyfill';
 import 'source-map-support/register';
 
-import debug from 'debug';
+import debugLog from 'debug';
 import fs from 'graceful-fs';
 import mkdirp from 'mkdirp';
```

Comment:
For logging within generic Node modules, [`debug`](https://www.npmjs.com/package/debug) is pretty much the defacto. The problem with forcing something like 'Winston` or `Bunyan` is that you're now creating a dependency on the Open Source consumer of this module (which could be anyone in the world) to use the logging paradigm we've chosen for our Linux player.